### PR TITLE
Feature mes 1268 global fonts

### PR DESCRIPTION
**Pull request by inhand**
## Description and relevant Jira numbers
MES-1268 Fonts, colors - all the global UI design needs
Added fonts, gds colours and mixins

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [ ] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [x] Reviewers selected in Github
- [x] Any new reusable component/widget added to component library on Confluence

- [x] Screenshot(s) captured on the physical device (if applicable)
- [x] Tested by QA
- [ ] PO's approval

### CODE REVIEW

**Comment by inhand** [comment]
This is an example of the classes and how they look.

screen shot 2018-12-14 at 09 26 27

screen shot 2018-12-14 at 09 29 18

File: src/pages/journal/journal.html [hunk 1]
```diff
@@ -14,6 +14,88 @@
     <ion-card-content>
       <button ion-button navPush="ContactDetailsPage">Contact Details</button>
       <button ion-button navPush="WaitingRoomPage">Waiting Room</button>
+
+      <div class="mes-text-bold-black-xxlarge"> .mes-text-bold-black-xxlarge </div>
```
Comment (szabolcs-keresztes) [inline src/pages/journal/journal.html]: Should we use h1 instead?

Comment (inhand) [reply src/pages/journal/journal.html]: Yes those divs are just to give me a visual. They will be removed when the PR is finalised

Comment (szabolcs-keresztes) [reply src/pages/journal/journal.html]: Sure. Just saying that styling up h1 globally could replace that div with that long class name

Comment (inhand) [reply src/pages/journal/journal.html]: Yep that can be done at a component level if we want as a h1 in one component may not be the same in another. We are trying to align the names with Abstract so we all know which styling we are talking about.

Comment (szabolcs-keresztes) [reply src/pages/journal/journal.html]: My intuition says that we should use the HTML elements in a global level, and have a unique h1, h2, h3, p etc. across the application. And these will represent the same concepts different types of headers paragraphs on all pages. So we'll be able to reference them in a conversation/documentation.
What do you think?

**Comment by szabolcs-keresztes** [comment]
Nice one, looks pretty slick with the universal HTML elements.

**Comment by inhand** [comment]
screen shot 2018-12-18 at 10 07 51

**Pull request merged**